src/NGen.cpp

```diff
@@ -57,7 +58,97 @@ int mpi_num_procs;
 
 std::unordered_map<std::string, std::ofstream> nexus_outfiles;
 
+void ngen::exec_info::runtime_summary(std::ostream& stream) noexcept
+{
+    stream << "Runtime configuration summary:\n";
+
+#if NGEN_WITH_MPI
+    stream << "  MPI:\n"
+           << "    Rank: " << mpi_rank << "\n"
+           << "    Processors: " << mpi_num_procs << "\n";
+#endif // NGEN_WITH_MPI
+  
+#if NGEN_WITH_PYTHON // -------------------------------------------------------
+    { // START RAII
+        py::scoped_interpreter guard{};
+
+        auto sys       = py::module_::import("sys");
+        auto sysconfig = py::module_::import("sysconfig");
+
+        // try catch
+        py::module_ numpy;
+        bool imported_numpy = false;
+        const char* err = nullptr;
+        try {
+            numpy = py::module_::import("numpy");
+            imported_numpy = true;
+        } catch(py::error_already_set& e) {
+            err = e.what();
```

Comment:
Does `err` become a dangling pointer once `e` goes out of scope on the next line? To spare thinking about this, let's just declare `std::string err`

Reply:
Fixed in [f504688](https://github.com/NOAA-OWP/ngen/pull/679/commits/f5046880013a80019a94aba2075a6925d2df46ad).